### ISSUE TITLE
Find new tools or techniques

### ISSUE THREAD

**Issue by lmangall** [ft-transcendence-inc/ft_transcendence#3]
explore as in stance https://docs.github.com/en/issues and find something we are not using yet but could be using 

@552020  

(this issue has been created through the _issue's tracking_ project from our github transcendance repo

**Comment by lmangall** [comment]
@552020  from the project in our repo : https://github.com/orgs/ft-transcendence-inc/projects/1/views/1?pane=issue&itemId=64735162

I couldn't tag Daniil, any idea ?

**Comment by 552020** [comment]
> @552020 from the project in our repo : https://github.com/orgs/ft-transcendence-inc/projects/1/views/1?pane=issue&itemId=64735162
> 
> I couldn't tag Daniil, any idea ?

Yes, he needs to accept invitation to the organisation and to the repo.

**Comment by lmangall** [comment]
Milestones something we haven't been using:
https://docs.github.com/en/issues/tracking-your-work-with-issues/quickstart#adding-milestones
